Shared/zh-Hans.lproj/Localizable.strings

```diff
@@ -138,6 +138,15 @@
 
 "SAVE_TO_PHOTOS" = "保存到照片";
 
+// History
+"HISTORY" = "历史";
+"%@_AGO" = "%@ ago"; // x 天前
+"REMOVE_CHAPTER_HISTORY_TEXT" = "这将删除此章节历史记录。是否确实要继续?";
```

Comment:
```suggestion
"REMOVE_CHAPTER_HISTORY_TEXT" = "这将删除此章节历史记录。是否确实要继续？";
```
You should keep the question mark variant consistent, make it either `U+FE56 ？SMALL QUESTION MARK` or `U+003F ? QUESTION MARK`

Reply:
If you decide to go with the former, you should also fix `RESTORE_BACKUP`'s question mark.